modules/administration-guide/pages/importing-untrusted-tls-certificates.adoc

```diff
@@ -7,29 +7,23 @@
 [id="importing-untrusted-tls-certificates_{context}"]
 = Importing untrusted TLS certificates to {prod-short}
 
-External communications between {prod-short} components are, by default, encrypted with TLS. Communications of {prod-short} components with external services such as proxies, source code repositories, and identity provider may require using TLS. Those communications require the use of TLS certificates signed by trusted Certificate Authorities.
+By default, external communications between {prod-short} components are encrypted with TLS. Communications of {prod-short} components with external services such as proxies, source code repositories, and identity provider might also require TLS. All communications encrypted with TLS require the use of TLS certificates signed by trusted Certificate Authorities (CA).
 
-NOTE: When the certificates used by {prod-short} components or by an external service are signed by an untrusted CA, it can be necessary to import the CA certificate in the {prod-short} installation so that every {prod-short} component will consider them as signed by a trusted CA.
-
-Typical cases that may require this addition are:
-
-* When the underlying {orch-name} cluster that uses TLS certificates signed by a CA that is not trusted.
-* When {prod-short} server or workspace components connect to external services such as {identity-provider} or a Git server that use TLS certificates signed by an untrusted CA.
+When the certificates used by {prod-short} components or by an external service are signed by an untrusted CA, it's necessary to import the CA certificate in the {prod-short} installation so that every {prod-short} component considers them as signed by a trusted CA. Following cases typically require this addition:
+* The underlying {orch-name} cluster uses TLS certificates signed by a CA that isn't trusted.
```

Comment:
Am I understanding this bullet list right? I'm reading it as a list of cases where you NEED to make sure it's signed by a trusted CA. Therefore, if it is NOT, you must import the trusted CA signature (like you stated above)? 

For minimalism, I think this sentence could be re-phrased, what do you think of this?: 

`The following cases require trusted CA authorization:

- The underlying {orch-name} cluster that uses TLS certificates
- {prod-short} server or workspace components that connect to external services such as {identity-provider} or a Git server that use TLS certificates`

Reply:
I think it's OK to keep it as it is because the current phrasing takes into account that the components might be using trusted certificates from the get-go.